packages/paste-website/src/pages/tokens/theme-package.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: Theme Package
+package: '@twilio-paste/theme'
+description: The Paste Theme Package allows easy and simple access to the design tokens via a theme provider to React components
+---
+
+import {graphql} from 'gatsby';
+import {Box} from '@twilio-paste/box';
+import Changelog from '@twilio-paste/theme/CHANGELOG.md';
+import {SidebarCategoryRoutes} from '../../constants';
+
+export const pageQuery = graphql`
+  {
+    allPasteThemePackage(filter: {name: {eq: "@twilio-paste/theme"}}) {
+      edges {
+        node {
+          name
+          description
+          version
+        }
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Theme package"
+  categoryRoute={SidebarCategoryRoutes.TOKENS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/components/button"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/overview-theme--provider"
+  data={props.data.allPasteThemePackage.edges}
+/>
+
+***
+
+<content>
+
+## Theme Provider
+
+Paste Components require you to wrap your application in the Paste Theme provider. This supplies Paste components with design token values needed to render each component, and provides a way to access those token values in your custom components.
+
+The Paste Theme Provider is a wrapper around the [Emotion Theme Provider](https://emotion.sh/docs/emotion-theming) and comes with a default theme.
+
+You should place the `Theme.Provider` around the root of your React application.
+
+```js
+import {Theme} from '@twilio-paste/theme';
+
+<Theme.Provider>
+  ...
+</Theme.Provider>
+```
+
+## Using tokens in custom components
+
+Sometimes you will need to create something custom that is not available in Paste but you need that component to still look like it's from Paste. The use of Paste Tokens in your styles is the way that you can access global design properties used in all Paste Components.
+
+### Emotion Styled Components (recommended)
+
+By using the `Theme.Provider`, when you create a custom component using Emotion CSS, the theme object is available on `props` via context. You can access those via the [Styled-System `themeGet` utility](https://styled-system.com/api#themeget) or props object directly.
+
+```js
+import styled from '@emotion/styled';
+import { themeGet } from 'styled-system';
+
+const custom = styled.div`
+  background-color: ${themeGet('backgrounds.primary')};
+  padding: ${props => props.theme.spacing.spacing20};
+`
+```
+
+### Alternatives when not using Emotion
+
+If you are not using Emotion styled components or want to access the values of tokens in your React components outside of styling, there are 3 ways this can be achieved.
```

Comment:
nit: swap 3 with three. We follow AP styleguide editorially (which Kyle's Tone of Voice will cover), so all numbers till and including nine will be spelled in editorial context. 10 and above will be numerical.